src/components/calcite-shell-floating-panel/calcite-shell-floating-panel.tsx

```diff
@@ -1,30 +1,10 @@
-import {
-  Component,
-  Element,
-  Event,
-  EventEmitter,
-  Host,
-  Prop,
-  State,
-  Watch,
-  h
-} from "@stencil/core";
-
-import { CalcitePlacement } from "../interfaces";
-
-import { getOffsetTop } from "../utils/position";
+import { Component, Element, Event, EventEmitter, Host, Prop, h } from "@stencil/core";
 
 import { x16 } from "@esri/calcite-ui-icons";
 
 import CalciteIcon from "../utils/CalciteIcon";
 
-import { CSS } from "./resources";
-
-import classnames from "classnames";
-
-import { getElementDir } from "../utils/dom";
-
-import { CSS_UTILITY } from "../utils/resources";
+import { CSS, TEXT } from "./resources";
 
 @Component({
   tag: "calcite-shell-floating-panel",
```

Comment:
This panel doesn't float anymore, the floating is achieved via the popover. Probably should rename.

Reply:
Outside of PR scope but valid. @asangma can think of a better name. Maybe more relative to what it is like a small or temporary panel.